posts/2024-05-13-shinyMobile-2.0.0/index.qmd

```diff
@@ -0,0 +1,1865 @@
+      ),
+      verbatimTextOutput("form")
+    )
+  ),
+  server = function(input, output, session) {
+    output$form <- renderPrint(input$myform)
+
+    observeEvent(input$update, {
+      updateF7Form(
+        "myform",
+        data = list(
+          "text" = "New text",
+          "textarea" = "New text area",
+          "password" = "New password"
+        )
+      )
+    })
+  }
+)
+```
+:::
+
+::: {.column width="25%"}
+
+:::
+
+::::
+
+```{r, eval=FALSE}
+library(shiny)
+library(shinyMobile)
+
+shinyApp(
+  ui = f7Page(
+    f7SingleLayout(
+      navbar = f7Navbar(title = "Inputs form"),
+      f7Block(f7Button("update", "Click me")),
+      f7BlockTitle("A list of inputs in a form"),
+      f7List(
+        inset = TRUE,
+        dividers = FALSE,
+        strong = TRUE,
+        f7Form(
+          id = "myform",
+          f7Text(
+            inputId = "text",
+            label = "Text input",
+            value = "Some text",
+            placeholder = "Your text here",
+            style = list(
+              description = "A cool text input",
+              outline = TRUE,
+              media = f7Icon("house"),
+              clearable = TRUE,
+              floating = TRUE
+            )
+          ),
+          f7TextArea(
+            inputId = "textarea",
+            label = "Text Area",
+            value = "Lorem ipsum dolor sit amet, consectetur
+              adipiscing elit, sed do eiusmod tempor incididunt ut
+              labore et dolore magna aliqua",
+            placeholder = "Your text here",
+            resize = TRUE,
+            style = list(
+              description = "A cool text input",
+              outline = TRUE,
+              media = f7Icon("house"),
+              clearable = TRUE,
+              floating = TRUE
+            )
+          ),
+          f7Password(
+            inputId = "password",
+            label = "Password:",
+            placeholder = "Your password here",
+            style = list(
+              description = "A cool text input",
+              outline = TRUE,
+              media = f7Icon("house"),
+              clearable = TRUE,
+              floating = TRUE
+            )
+          )
+        )
+      ),
+      verbatimTextOutput("form")
+    )
+  ),
+  server = function(input, output, session) {
+    output$form <- renderPrint(input$myform)
+
+    observeEvent(input$update, {
+      updateF7Form(
+        "myform",
+        data = list(
+          "text" = "New text",
+          "textarea" = "New text area",
+          "password" = "New password"
+        )
+      )
+    })
+  }
+)
+```
+
+## Breaking changes
+
+Some components have disappeared from Framework7 and we we had to deprecate them as they no longer work. Other long time deprecated `{shinyMobile}` elements are also removed to cleanup the codebase. We invite you to review the [changelog](https://github.com/RinteRface/shinyMobile/blob/69da6ca46984bf6c73e2dc32ff8d9f415ec36a30/NEWS.md#breaking-changes).
```

Comment:
Correction: remove the double "we" from line 1.

Suggestion: "We invite you to review the [changelog](https://github.com/RinteRface/shinyMobile/blob/69da6ca46984bf6c73e2dc32ff8d9f415ec36a30/NEWS.md#breaking-changes) **to see a list of all changes in this release**."